docs/source-2.0/additional-specs/smoke-tests.rst

```diff
@@ -0,0 +1,194 @@
+.. _smoke-tests:
+
+===========
+Smoke Tests
+===========
+
+Smoke tests are small, simple tests intended to uncover large issues by ensuring
+basic functionality works as expected. In Smithy, these tests are used to make
+sure clients can successfully make a request to a live service, and that the
+service responds as expected.
+
+
+--------
+Overview
+--------
+
+This specification defines a single trait in the ``smithy.test`` namespace that is
+used to make basic assertions about how the service should respond to a given
+request from the client.
+
+:ref:`smithy.test#smokeTests <smokeTests-trait>`
+    Defines a set of test cases to send to a live service to ensure that a
```

Comment:
nit: is `live` needed here?

Reply:
Maybe? This contrasts with, for example, protocol tests which don't run against live services. Maybe not entirely necessary but I don't think it hurts to be more specific here